reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -600,6 +600,17 @@ public Emission getReason() {
 		@Deprecated
 		void emitError(Throwable error);
 
+		/**
+		 * Determines how many active {@link Subscriber Subscribers} the sink currently has.
```

Comment:
should we maybe define "active" here? Otherwise, someone may think that "active" == "ready to consume / requested", which isn't the case

Reply:
`current` sounds less ambiguous, I'll change to that and explain a bit more in the javadoc